evenOccurrences/evenOccurrences.js

```diff
@@ -10,6 +10,16 @@
  *    * console.log(onlyEven); //  4
  *    */
 
+
+
 const evenOccurence = (arr) => {
   // Your code here.
-};
\ No newline at end of file
+  const onlyEven = evenOccurence([1, 7, 2, 4, 5, 1, 6, 8, 9, 6, 4, 1]);
+  const arr2 = [];
+  for (i = 0; i < arr.length; i++);
+  if(arr[i] % 2 === 0) {
```

Comment:
You should open and close your for loop block. It's just makes it read a little easier.